Cesium3DTilesContent/src/BatchTableToGltfStructuralMetadata.cpp

```diff
@@ -2073,38 +2071,45 @@ struct BatchIdSemantic {
       return;
     }
     uint32_t byteOffset = byteOffsetIt->value.GetUint();
+    MetadataProperty::ComponentType componentType;
     const auto componentTypeIt = batchIdIt->value.FindMember("componentType");
-    if (componentTypeIt != batchIdIt->value.MemberEnd() &&
-        componentTypeIt->value.IsString()) {
+    if (componentTypeIt == batchIdIt->value.MemberEnd()) {
+      componentType = MetadataProperty::ComponentType::UNSIGNED_SHORT;
+    } else {
```

Comment:
NITPICK: I think this could be condensed further. You would still return early if the `componentType` property is incorrectly formed:

```c++
    MetadataProperty::ComponentType componentType = MetadataProperty::ComponentType::UNSIGNED_SHORT;

    const auto componentTypeIt = batchIdIt->value.FindMember("componentType");
    if (componentTypeIt != batchIdIt->value.MemberEnd()) {